EPAM_iOS_School_Practice_2_Star_Wars/EPAM_iOS_School_Practice_2_Star_Wars/Networking/SearchService.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  SearchService.swift
+//  EPAM_iOS_School_Practice_2_Star_Wars
+//
+//  Created by Maxim on 27.02.2020.
+//  Copyright © 2020 Maksim Svyatov. All rights reserved.
+//
+
+import Foundation
+import Alamofire
+
+protocol SearchServiceProtocol {
+    func searchPeople(for name: String, completion: @escaping ((People?, Error?)->()))
+}
+
+class SearchService: SearchServiceProtocol {
```

Comment:
Would be better: `NetworkManager` instead `SearchService`

Reply:
Fixed